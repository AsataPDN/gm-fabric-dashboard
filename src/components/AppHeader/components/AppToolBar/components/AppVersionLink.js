import styled from "styled-components";

import { spacingScale } from "../../../../../style/styleFunctions";
import { FONT_SIZE_XS } from "../../../../../style/styleVariables";

const AppVersionLink = styled.a`
  font-weight: 600;
  font-size: ${FONT_SIZE_XS};
  flex: 0 0 auto;
  opacity: 1;
  padding: 0 ${spacingScale(1)};
  color: white;

  &:hover {
    color: white;
    opacity: 1;
  }
`;

export default AppVersionLink;
