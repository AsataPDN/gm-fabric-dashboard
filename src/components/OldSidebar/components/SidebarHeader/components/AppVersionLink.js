import styled from "styled-components";

const AppVersionLink = styled.a`
  font-weight: 600;
  font-size: 11px;
  flex: 0 0 auto;
  opacity: 0.5;
  padding: 0 8px;
  color: white;

  &:hover {
    color: white;
    opacity: 1;
  }
`;

export default AppVersionLink;
