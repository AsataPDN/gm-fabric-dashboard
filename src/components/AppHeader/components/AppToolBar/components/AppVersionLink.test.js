import React from "react";
import { shallow } from "enzyme";
import AppVersionLink from "./AppVersionLink";

describe("AppVersionLink", () => {
  it("matches snapshot", () => {
    const aAppVersionLink = shallow(<AppVersionLink />);
    expect(aAppVersionLink).toMatchSnapshot();
  });
});
