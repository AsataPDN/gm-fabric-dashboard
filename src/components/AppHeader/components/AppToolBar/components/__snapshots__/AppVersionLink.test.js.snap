// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AppVersionLink matches snapshot 1`] = `
.c0 {
  color: white;
  -webkit-flex: 0 0 auto;
  -ms-flex: 0 0 auto;
  flex: 0 0 auto;
  font-size: 9px;
  font-weight: 600;
  opacity: 1;
  padding: 0 8px;
}

.c0:hover {
  color: white;
  opacity: 1;
}

<a
  className="c0"
/>
`;
